javascript/src/main/App.js

```diff
@@ -39,6 +39,7 @@ function App() {
   return (
     <div className="App">
       <AppNavbar />
+      <div class="flex-wrapper">
```

Comment:
Instead of creating a new css class, can we use existing Bootstrap utilities? ([docs](https://getbootstrap.com/docs/4.0/utilities/flex/))

Reply:
I agree with Bryan, but I think we may want to make that be a separate issue given the date :-)